reconcile/aus/base.py

```diff
@@ -1002,16 +1001,31 @@ def set_mutex(
                     )
                 )
             else:
+                target_version_prefix = get_version_prefix(version)
+                minor_version_gates = gates_for_minor_version(
+                    gates, target_version_prefix
+                )
+                # skipping upgrades when there are no version gates is a safety
+                # precaution to prevent cluster upgrades being scheduled.
+                # missing version gates are an indicator that the version has not yet gone
+                # through SREP gap analysis and is not yet ready for upgrades.
+                #
+                # this might change in the future - revisite for 4.16
+                if not minor_version_gates:
+                    logging.warning(
```

Comment:
I wonder if warning is the best log level here. Will it continuously write in the reconcile channel?

Reply:
you are right, it will continously write there. lets go for debug for the time being.